Example/TABTestKit/BiometricLoginViewController.swift

```diff
@@ -26,10 +26,12 @@ final class BiometricLoginViewController: UIViewController {
     let canUseBiometrics = context.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: .init(nilLiteral: ()))
     if canUseBiometrics {
       context.evaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, localizedReason: "Log in", reply: { [weak self] wasSuccessful, error in
-        if wasSuccessful {
-          self?.handleSuccessfulLogin()
-        } else {
-          self?.handleBiometricsFailed()
+        DispatchQueue.main.async {
```

Comment:
This was crashing because the `reply` callback was being called on a background thread which crashes UIKit, so bumping onto the main queue here fixes that